### PR TITLE
Load outbrain instantly on international edition

### PR DESCRIPTION
**Pull request by uplne**
There is no high relevance commercial component on international edition so we don't need to wait for DFP to return.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ define([
                 && !config.page.isPreview
                 && this.identityPolicy()
                 && config.page.section !== 'childrens-books-site') {
-                if (detect.adblockInUse()) {
+                if (detect.adblockInUse() || config.page.edition.toLowerCase() === 'int') {
```
Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js]: I'm not sure if it's explicit, looking at this code, _why_ we can load instantly on the international edition. Maybe an inner function like `hasHighRelevanceComponent` might be more obvious? Or at least a note in the test?

Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js]: Good idea

**Comment by uplne** [comment]
@jbreckmckye I've updated the naming

**Comment by jbreckmckye** [comment]
Great. :+1:

**Pull request merged**